CSharp/App/Program.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using Google.Cloud.Firestore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+FirestoreDb db = FirestoreDb.Create("cold-start-417102");
+int counter = 0;
+
+// Everytime the user hits the /hello endpoint, the counter will increment by 1
+app.MapGet("/hello", () => {
+    counter++;

```

Comment:
Use [Interlocked.Increment](https://learn.microsoft.com/en-us/dotnet/api/system.threading.interlocked.increment?view=net-8.0) to avoid a [race condition](https://en.wikipedia.org/wiki/Race_condition#Example).